UserShiftsApiService/UserShiftsApiService/Entities/UserDateRangePreferenceRequestEntity.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace UserShiftsApiService.Entities;
+
+public class UserDateRangePreferenceRequestEntity
+{
+    public string Id { get; set; } 
+    public DateTime StartingDate { get; set; }
+    public DateTime EndingDate { get; set; }
+    public DateRangeRequestType RequestType { get; set; }
+    
+    public string UserId { get; set; }
+    
+    public UserEntity User { get; set; }
```

Comment:
its a cool convention to make these props `virtual` 
it doesnt do anything but makes it clear they should not be set